spark-streaming-sql/query.sql

```diff
@@ -0,0 +1,59 @@
+WITH raw AS (
+  SELECT 
+    poc_id,
+    date,
+    beacon_report,
+    explode_outer(selected_witnesses) as witness,
+    true as selected
+    FROM lora_poc_v1
+  UNION ALL 
+  SELECT poc_id,
+    date,
+    beacon_report,
+    explode_outer(unselected_witnesses) as witness,
+    false as selected
+  FROM lora_poc_v1
+)
+SELECT
+    date,
+    selected,
+    base64(poc_id) as poc_id,
+    struct(
+      from_unixtime(beacon_report.received_timestamp / 1000) as received_timestamp_millis,
+      cast(beacon_report.location as decimal(23, 0)) location,
+      (beacon_report.hex_scale / 10000) as scale,
+      b58encodeChecked(beacon_report.report.pub_key) as pub_key,
+      base64(beacon_report.report.local_entropy) as local_entropy,
+      base64(beacon_report.report.remote_entropy) as remote_entropy,
+      base64(beacon_report.report.data) as data,
+      beacon_report.report.timestamp as timestamp_nanos,
+      beacon_report.report.tmst as tmst,
```

Comment:
You've discovered that `tmst` is a weird internal clock counter on the LoRa concentrator, I'm sure. As such, I think we should explicitly name the output column `concentrator_tmst`.